Modules/timemodule.c

```diff
@@ -1736,27 +1736,9 @@ if it is 1, the time is given in the DST time zone;\n\
 if it is -1, mktime() should guess based on the date and time.\n");
 
 
-
-static struct PyModuleDef timemodule = {
-    PyModuleDef_HEAD_INIT,
-    "time",
-    module_doc,
-    -1,
-    time_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-PyMODINIT_FUNC
-PyInit_time(void)
+static int
+time_exec(PyObject *m)
```

Comment:
Use `mod` or `module` would be more exact than `m` ;)

Reply:
Yeah, please this PR as an opportunity to replace m with module ;-)

Reply:
I have an impractical idea: Making an review robot to give some suggestions of normative problem automatically ;)